src/fldeff_flash.c

```diff
@@ -28,38 +29,38 @@ struct FlashStruct
 // static functions
 static void hm2_flash(void);
 static void sub_81371B4(void);
-static bool8 sub_8137304(void);
-static void sub_81373F0(void);
-static void sub_8137404(u8 taskId);
-static void sub_8137420(u8 taskId);
-static void sub_81374C4(u8 taskId);
-static void sub_813750C(u8 taskId);
-static void sub_8137574(u8 taskId);
-static void sub_81375A8(void);
-static void sub_81375BC(u8 taskId);
-static void sub_81375D8(u8 taskId);
-static void sub_8137678(u8 taskId);
-static void sub_81376DC(u8 taskId);
+static bool8 MaybeDoMapTransition(void);
```

Comment:
What about TryDoMapTransition? MaybeDoMapTransition sounds like we're not sure what it does